rfcs/expected_fail_message_tag_for_subtests.md

```diff
@@ -0,0 +1,71 @@
+# RFC 165: Support expected-fail-message in test metadata for subtests
+
+## Summary
+
+Add "expected-fail-message" in test metadata so that we can track how
+a subtest fails.
+
+## Details
+
+In Chromium developers are using baselines to track expected output message
+for testharness tests. While the output message for a pass subtest is
```

Comment:
Is there even a message anywhere when passing?

Reply:
We do not include the message in -expected.txt for a pass subtest, but the field is always there from Wptrunner's point of view. Any suggestion for rewording if needed?